code/style.css

```diff
@@ -0,0 +1,142 @@
+html,
+body {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: white;
+  color: whitesmoke;
+  font-family: "futura", sans-serif; 
+}
+
+.main-container {
+  display: flex;
+  width: 100%;
+}
+
+img {
+  width: 50%;
+  object-fit: cover;
+}
+
+.form-container {
+  background-color: #52776f;
+  width: 50%;
+}
+
+.content-container {
+  margin: 70px;
+}
+  
+.content-text {
+  text-align: center;
+}
+
+.content-text h1 {
+  font-size: 60px;
+}
+
+.content-text p {
+  font-size: 25px;
+}
+
+legend {
+  font-size: 20px;
+}
+
+fieldset {
+  border-radius: 6px;
+}
+
+.form-action-group {
+  margin: 20px;
+}
+
+.form-action-group input {
+  width: 100%;
+  height: 30px;
```

Comment:
It's better to stretch our input fields by `font-size` (it will increase height) and `padding`. Try that out!